x-pack/plugins/observability_solution/apm/public/components/shared/entity_enablement/feedback_modal.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButton, EuiConfirmModal } from '@elastic/eui';
+
+export function FeedbackModal({
+  isFeedbackModalVisiable,
+  setsIsFeedbackModalVisiable,
+  refetch,
+}: {
+  isFeedbackModalVisiable: boolean;
+  setsIsFeedbackModalVisiable: (value: boolean) => void;
+  refetch: () => void;
```

Comment:
Suggestion: declaring boolean props optional and setting a default value can make the component more user-friendly. Consumers would pass a value only when they want to override the default behavior. 

Instead of `setsIsFeedbackModalVisiable` I would provide an event handler so that the consumer can decide what to do when the modal is closed. wdyt?

```suggestion
  isFeedbackModalVisiable = false,
  onClose,
}: {
  isFeedbackModalVisiable?: boolean;
  onClose: () => void;
```

Reply:
Great suggestion, let me update it.